src/controllers/voteController.js

```diff
@@ -1,33 +1,40 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable indent */
+import dotenv from 'dotenv';
 import voteService from '../services/VoteService';
 import PetitionService from '../services/PetitionServices';
 import Util from '../utils/Utils';
+import db from '../models';
 
 const util = new Util();
 
 class voteController {
   // eslint-disable-next-line consistent-return
   static async addVote(req, res) {
-        if (!req.body.user_id || !req.body.vote || !req.params) {
+        if (!req.body.vote || !req.params) {
           util.setError(400, 'Please provide complete details');
           return util.send(res);
         }
+        const petition = await db.petitions.findOne({ where: { id: Number(req.params.id) } });
+        if (!petition) {
+            util.setError(400, 'Petition does not exist');
+            return util.send(res);
+            }
         // eslint-disable-next-line camelcase
-        const { vote, user_id } = req.body;
+        const { vote } = req.body;
         // userId get from token
         const newId = req.params;
         // eslint-disable-next-line camelcase
         const petition_id = newId.id;
```

Comment:
There is redundancy in getting your user id, a better way might be like that, >>> const petition_id = req.params.id